postgres/src/main/java/org/jdbi/v3/postgres/PostgresPlugin.java

```diff
@@ -14,8 +14,10 @@
 package org.jdbi.v3.postgres;
 
 import java.sql.Connection;
+import java.sql.SQLException;
 import java.util.Map;
 
+import com.pgvector.PGvector;
```

Comment:
Is there a test that confirms this is still working with the dependency absent?

Reply:
`postgis`, `moshi`, `jackson`, `gson2` artifacts all depend on `jdbi3-postgresql but not `pgvector` and work ok, so I think that's sufficient? It is a little bit accidental but it does cover this case.